docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,199 @@
+
+- Getting battery usage information of an application.
+
+    You can [retrieve battery usage information of an application](#appusage_get) by specifying its application ID, [resource IDs](#resource_key), and the [time duration](#duration_key).
+
+- Getting battery usage information of a resource.
+
+    You can [retrieve battery usage information of single or multiple resources](#resourceusage_get) by specifying its [resource IDs](#resource_key), and the [time duration](#duration_key).
+
+## Prerequisites
+
+To enable your application to use the Battery Monitor functionality:
+
+1. To use the [Battery Monitor](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__MONITOR__MODULE.html) API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```xml
+    <privileges>
+       <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+    </privileges>
+    ```
+
+2. To use the functions and data types of the Battery Monitor API, include the `<battery_monitor.h>` header file in your application:
+
+    ```
+    #include <battery_monitor.h>
+    ```
+
+<a name="appusage_get"></a>
+## Getting Battery Usage Information of Application
+
+To get the battery usage information related with application ID:
+
+- You can get information about an application for a particular resource over a specific duration of time.
+  To get resource ID, use the `battery_monitor_resource_id_e` enum and to get the time duration type information, use the `battery_monitor_duration_type_e` enum:
+
+    ```
+    #include <battery_monitor.h>
+
+    void
+    func(void)
+    {
+       char* app_id = "org.tizen.samsung";
+
+       int error_code = BATTERY_MONITOR_ERROR_NONE;
+       int battery_usage = -1;
+       battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
+       battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+       error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+       if (error_code == BATTERY_MONITOR_ERROR_NONE)
+           printf("The Battery Usage for appid [%s], for resource display is [%d]", app_id, battery_usage);
+       else
+           printf("Error Occurred [%d]", error_code);
+
+       resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
+       duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+
+       error_code = battery_monitor_get_usage_by_app_id_for_resource_id(app_id, resource_id, duration_val, &battery_usage);
+
+       if (error_code == BATTERY_MONITOR_ERROR_NONE)
+           printf("The Battery Usage for appid [%s], for resource wifi is [%d]", app_id, battery_usage);
+
+       else
+           printf("Error Occurred [%d]", error_code);
+
+       free(app_id);
+
+       return;
+    }
+    ```
+
+- You can get information on total battery usage of an application ID by combining all the resources over a specific duration of time.
```

Comment:
Thanks @mijin85cho 
Can I change this sentence to:
Get total battery usage of an application ID over a specific duration of time.
As it conveys the API functioning more effectively.
Since it not for a particular resource, its cumulative of all the resources used by the application.

Reply:
@d1-agarwal 
As suggested by @mijin85cho in 
[Comment 1](https://github.com/Samsung/tizen-docs/pull/847/files#r334312264)
[Comment 2](https://github.com/Samsung/tizen-docs/pull/847/files#r334313137)
I have rephrased it to the following final form: 

Request  you to change lines 82-83 to : 

- Get information about the total battery usage of an application ID with all its resources combined over a specific duration of time.
   
     To get a specific duration type, use the `battery_monitor_duration_type_e` enum:

Reply:
@d1-agarwal Ok, I agree with your sentence.

Get total battery usage of an application ID over a specific duration of time.

Reply:
@mijin85cho 
Sorry but I think you have mistakenly posted it here, as the sentence is: 
**Original:** You can get information on total battery usage of an application ID by combining all the resources over a specific duration of time.

**You suggested:** 

Get battery usage information of an application ID for a particular resource over a specific duration of time.

It is wrong in the context of the original meaning of the sentence.